cloudflare/resource_cloudflare_waf_group.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
 )
 
+const CLOUDFLARE_INVALID_OR_REMOVED_WAF_RULE_SET_ID_ERROR = 1003
```

Comment:
Should I put those into their own file or is that fine to have per-file constants here ?

Reply:
Happy to start here and we can always extract them out and DRY them up if needed.